src/components/Search/Search.tsx

```diff
@@ -108,92 +108,94 @@ const Search = ({
+        ref={searchContainerRef}
       >
-        <form
-          data-testid={`${baseTestId}-form`}
-          className={classnames(`${baseClassName}__form`, {
-            [`${baseClassName}__form--active`]: isSearchExpanded,
-          })}
-          ref={searchFormRef}
+        <Text variant={TextVariants.heading4} className={`${baseClassName}__container__inner__label`}>
+          {searchButtonText}
+        </Text>
+        <div
+          {...commonProps}
+          className={classnames(baseClassName, className, { [`${baseClassName}--active`]: isSearchExpanded })}
+          data-testid={baseTestId}
+          role="search"
+          {...props}
         >
-          <div
-            className={classnames(`${baseClassName}__content-wrapper`, {
-              [`${baseClassName}__content-wrapper--active`]: isSearchExpanded,
+          <form
+            data-testid={`${baseTestId}-form`}
+            className={classnames(`${baseClassName}__form`, {
+              [`${baseClassName}__form--active`]: isSearchExpanded,
             })}
-            role="combobox"
-            aria-haspopup="listbox"
+            ref={searchFormRef}
           >
-            <CSSTransition
-              in={isSearchExpanded}
-              classNames={`${px}-input`}
-              addEndListener={() => {
-                return;
-              }}
+            <div
+              className={classnames(`${baseClassName}__content-wrapper`, {
+                [`${baseClassName}__content-wrapper--active`]: isSearchExpanded,
+              })}
+              role="combobox"
+              aria-haspopup="listbox"
             >
-              <Input
-                aria-hidden={!isSearchExpanded}
-                className={`${baseClassName}__input`}
-                id="search-input"
-                hideLabel
-                labelText={searchButtonText}
-                placeholder={isSearchExpanded ? placeholder : ''}
-                type="text"
-                defaultValue={defaultValue}
-                invalid={state === 'invalid'}
-                invalidText={invalidText}
-                onKeyDown={onKeyDown}
-                onChange={
-                  onSearch
-                    ? (e: { target: { value: string } }) => {
-                        onSearch(e.target.value);
-                      }
-                    : undefined
-                }
-                ref={searchInputRef}
+              <CSSTransition
+                in={isSearchExpanded}
+                classNames={`${px}-input`}
+                addEndListener={() => {
+                  return;
+                }}
+              >
+                <Input
+                  aria-hidden={!isSearchExpanded}
+                  className={`${baseClassName}__input`}
+                  id="search-input"
+                  hideLabel
+                  labelText={searchButtonText}
+                  placeholder={isSearchExpanded ? placeholder : ''}
```

Comment:
This is what we implemented when we were pairing. It's to avoid the weird slide under that was happening when the input was sliding back to being hidden.

Reply:
ahhh. I was reading it as the inverse. Yeah I remember now.